Porting FxCop Rules to Roslyn.md

```diff
@@ -0,0 +1,66 @@
+# Porting Managed Code Analysis Rules to Roslyn
+
+Visual Studio 2015 shipped with over 300 Code Analysis rules for managed code. These rules were all written as binary analysis rules using the FxCop engine. We are now engaged in an effort to rewrite some of these rules as Roslyn analyzers. This has the following benefits:
```

Comment:
It is probably better to note that FxCop operates against MSIL. 'binary analysis' is a bit generic, since this can refer to portable executable checkers (such as binskim). Historically, the IL-driven approach was very valuable, because we could analyze multiple managed assembly producers (including VB, C# and managed C++) with a single tool. Today, Roslyn covers C# and VB and provides important new benefits, 'while you type' detection and fixers being most prominent as you note.